repos/spack_repo/builtin/packages/moab/package.py

```diff
@@ -64,6 +65,8 @@ class Moab(AutotoolsPackage):
     conflicts("+pnetcdf", when="~mpi")
     conflicts("+parmetis", when="~mpi")
     conflicts("+coupler", when="~mpi")
+    conflicts("+tempest", when="~netcdf")
+    conflicts("+tempest", when="@:5.0.2")
```

Comment:
I'd recommend making 5.1.0 (or 5.5.0) the minimum requirement here if you want to support only the latest changes.

Reply:
I think this says that we don't support 5.0.2 or older.  I think if we put 5.1.0 here, it indicates that we *don't* allow that version.

We could switch this to 5.4.1 if we don't want to include tempest-remap support versions older than 5.5.0.  Presumably, folks won't really want those older versions.

Reply:
This might be clearer:
```suggestion
    # The 'tempest' variant is only supported starting from version 5.1.0
    conflicts("+tempest", when="@:5.0.99")
```